configs/KAKUTEF7MINI/config.h

```diff
@@ -81,29 +82,26 @@
 #define USB_DETECT_PIN       PA8
 
 #define TIMER_PIN_MAPPING \
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0) \
-    TIMER_PIN_MAP( 8, PB3 , 1,  0)
+    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  1) \
```

Comment:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  -1) \
```
Same comment on removing RX_PPM

Reply:
Also applied to remaining F7 boards. :+1: